snekomatic/messaging.py

```diff
@@ -0,0 +1,57 @@
+"""Messaging for the snekomatic app"""
+# dedent, remove single newlines (but not double-newlines), remove
+# leading/trailing newlines
+
+
+def _fix_markdown(s):
+    import textwrap
+
+    s = s.strip("\n")
+    s = textwrap.dedent(s)
+    s = s.replace("\n\n", "__PARAGRAPH_BREAK__")
+    s = s.replace("\n", " ")
+    s = s.replace("__PARAGRAPH_BREAK__", "\n\n")
+    return s
+
+
+invite_message = _fix_markdown(
```

Comment:
Could as well just have a Jinja template instead.

Reply:
That is a good point! I would like to add that in as well.